### PR TITLE
use the bootstrap icons

### LINKED ISSUE #5718: Support for Bootstrap v5?
Are there plans to support the latest bootstrap iteration (v5)? If so: Is there a roadmap?

### PR DESCRIPTION
**Pull request by UtechtDustin**
**Bug fix?**
no

**New Feature?**
no

**Resolve an issue?**
fix #5718

**Example(s)?**
Not possible @wenzhixin we need the newest bootstrap 5 version and the bootstrap icons for the editor.
Also we should now set bootstrap 5 as default on our examples  (https://examples.bootstrap-table.com/).

### CODE REVIEW

**Review by djhvscf — APPROVED** [review]

**Comment by wenzhixin** [comment]
OK, let me update the editor.

File: dist/bootstrap-table.js [hunk 1]
```diff
@@ -2813,7 +2813,7 @@
       }
     },
     5: {
-      iconsPrefix: 'fa',
+      iconsPrefix: 'bi',
```
Comment (wenzhixin) [inline dist/bootstrap-table.js:2816]: remove this change (`dist`)

**Comment by marceloverdijk** [comment]
@wenzhixin @UtechtDustin would it be possible to use `bootstrap-icons` for the sorting icons?
e.g using `arrow-up`, `arrow-up-short`, `sort-up` or any equivalent?

**Comment by wenzhixin** [comment]
> @wenzhixin @UtechtDustin would it be possible to use `bootstrap-icons` for the sorting icons? e.g using `arrow-up`, `arrow-up-short`, `sort-up` or any equivalent?

Not support this feature.

**Comment by marceloverdijk** [comment]
Np , thx for confirming.

**Pull request merged**